pallets/asset/src/lib.rs

```diff
@@ -475,6 +485,7 @@ decl_module! {
             };
             <Tokens<T>>::insert(&ticker, token);
             <BalanceOf<T>>::insert(ticker, did, total_supply);
+            Portfolio::<T>::set_default_portfolio_balance(did, &ticker, total_supply);
```

Comment:
Not really related to this PR, but tokens minted on asset creation should go to the treasuryDID rather than the identity that creates the asset (they may be the same).